scss/_variables.scss

```diff
@@ -44,7 +44,7 @@ $popup-padding-x: 0 !default;
 $popup-padding-y: 0 !default;
 
 $header-padding-x: $padding-x !default;
-$header-padding-y: $padding-y !default;
+$header-padding-y: $padding-x !default;
```

Comment:
I find this confusing in terms of variables -- changing `$padding-y` will have no effect on `$header-padding-y`, and changing `$padding-x` will. Why not `$header-padding-y: 2 * $padding-y;`? While the values are the same, it makes no sense to have a relationship between x and y variables.